save-cloud-charts/save-cloud/templates/orchestrator-service-account.yaml

```diff
@@ -18,6 +24,9 @@ rules:
     verbs: [list, get, delete, create]
   # this rule is required for sprint-cloud-kubernetes-fabric8-config
   - apiGroups: [""]  # "" indicates the core API group
+    resources: [pods/portforward]
```

Comment:
Guess it can be removed as it is not used anymore

Reply:
sure